orphedomos-maven-plugin/src/main/java/org/technologybrewery/orphedomos/mojo/DockerDeployAllMojo.java

```diff
@@ -0,0 +1,34 @@
+package org.technologybrewery.orphedomos.mojo;
+
+import org.technologybrewery.orphedomos.util.exec.DockerDeployExecutor;
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Mojo;
+import org.apache.maven.project.MavenProject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+
+@Mojo(name = "push-all-docker-images", aggregator = true)
+public class DockerDeployAllMojo extends AbstractDockerMojo {
+
+    private static final Logger logger = LoggerFactory.getLogger(DockerDeployAllMojo.class);
+
+    @Override
+    public void doExecute() throws MojoExecutionException {
+        login();
+        List<MavenProject> projects = mavenSession.getProjects();
+        for(MavenProject project : projects) {
+            getDockerDeployInfo(project, repoUrl).ifPresent((dockerDeployInfo) -> {
+                DockerDeployExecutor executor = new DockerDeployExecutor(project.getBasedir());
+                try {
+                    executor.deploy(dockerDeployInfo, aliases, dryRun);
```

Comment:
Q: I don't understand why this is necessary. Wouldn't you just run the build and the deploy would happen as a natural part of building the module? Why try to manually control the execution from the top-level project?  And in the context of populating an isolated environment, isn't it much much more relevant to push the base images of each image rather than the image itself?